hoomd/mpcd/__init__.py

```diff
@@ -171,9 +171,9 @@ def __init__(self, dt, aniso=None):
             self.set_params(aniso=aniso)
 
     _aniso_modes = {
-        None: _md.IntegratorAnisotropicMode.Automatic,
-        True: _md.IntegratorAnisotropicMode.Anisotropic,
-        False: _md.IntegratorAnisotropicMode.Isotropic
+        #None: _md.IntegratorAnisotropicMode.Automatic,
```

Comment:
This was a quick hack to get this to run. Will make an empty dict before merging, then this will be rewritten later anyway.